Develop an SQL schema for SOPs

```diff
@@ -1,29 +1,4351 @@
+SET xmloption = content;
+SET client_min_messages = warning;
+SET row_security = off;
+
+--
+-- TOC entry 10 (class 2615 OID 17146)
+-- Name: MzTab; Type: SCHEMA; Schema: -; Owner: postgres
+--
+
+CREATE SCHEMA "MzTab";
+
+
+ALTER SCHEMA "MzTab" OWNER TO postgres;
+
+--
+-- TOC entry 4 (class 2615 OID 17171)
+-- Name: MzTab-M; Type: SCHEMA; Schema: -; Owner: postgres
+--
+
+CREATE SCHEMA "MzTab-M";
+
+
+ALTER SCHEMA "MzTab-M" OWNER TO postgres;
+
+--
+-- TOC entry 8 (class 2615 OID 17329)
+-- Name: PQP; Type: SCHEMA; Schema: -; Owner: postgres
+--
+
+CREATE SCHEMA "PQP";
+
+
+ALTER SCHEMA "PQP" OWNER TO postgres;
+
+--
+-- TOC entry 11 (class 2615 OID 17316)
+-- Name: XML; Type: SCHEMA; Schema: -; Owner: postgres
+--
+
+CREATE SCHEMA "XML";
+
+
+ALTER SCHEMA "XML" OWNER TO postgres;
+
+SET default_tablespace = '';
+
+SET default_table_access_method = heap;
+
+--
+-- TOC entry 208 (class 1259 OID 17255)
+-- Name: Metadata_Proteomics; Type: TABLE; Schema: MzTab; Owner: postgres
+--
+
+CREATE TABLE "MzTab"."Metadata_Proteomics" (
+    "mzTab-version" text NOT NULL,
+    "mzTab-mode" text NOT NULL,
+    "mzTab-type" text NOT NULL,
+    "mzTab-ID" text NOT NULL,
+    title text,
+    description text,
+    sample_processing_number text,
+    sample_processing text,
+    instrument_number text,
+    instrument text,
+    "instrument-name" text,
+    "instrument-source" text,
+    analyzer_number text,
+    "instrument-analyzer" text,
+    "instrument-detector" text,
+    software_number text NOT NULL,
+    software text NOT NULL,
+    setting_number text,
+    "software-setting" text,
+    protein_search_engine_score_number text,
+    protein_search_engine_score text,
+    peptide_search_engine_score_number text,
+    peptide_search_engine_score text,
+    psm_search_engine_score_number text,
+    psm_search_engine_score text,
+    smallmolecule_search_engine_score_number text,
+    smallmolecule_search_engine_score text,
+    false_discovery_rate text[],
+    publication_number text,
+    publication text,
+    contact_number text,
+    "contact-name" text,
+    "contact-affiliation" text,
+    "contact-email" text,
+    uri_number text,
+    uri text,
+    fixed_mod_number text,
+    fixed_mod text,
+    "fixed_mod-site" text,
+    "fixed_mod-position" text,
+    variable_mod_number text,
+    variable_mod text,
+    "variable_mod-site" text,
+    "variable_mod-position" text,
+    quantification_method text NOT NULL,
+    "protein-quantification_unit" text,
+    "peptide-quantification_unit" text,
+    "small_molecule-quantification_unit" text,
+    ms_run_number text NOT NULL,
+    "ms_run-format" text,
+    "ms_run-location" text NOT NULL,
+    "ms_run-id_format" text,
+    fragmentation_method_number text NOT NULL,
+    "ms_run-fragmentation_method" text NOT NULL,
+    "ms_run-hash" text,
+    "ms_run-hash_method" text,
+    custom_number text,
+    custom text,
+    sample_number text,
+    species_number text,
```

Comment:
I am assuming these `_number` columns should also not be text, but perhaps integers?

Reply:
done